docs/source/conf.py

```diff
@@ -430,7 +430,7 @@
     ("py:class", "remove"),
     # sphinx can't resolve TYPE_CHECKING imports
     ("py:class", "LiveSpan"),
-    ("py:class", "RequestMessage"),
+    ("py:class", "ChatMessage"),
```

Comment:
note: Renaming `RequestMessage` to `ChatMessage` as it is used for both request and response messages.